### ISSUE TITLE
NTR: Dialeptic seizure

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5308]
"-Seizure
---Status epilepticus
---Infection-related
---Reflex seizure
---Motor seizure
---Focal-onset seizure
---Generalized-onset seizure
---Neonatal seizure
---NTR: Dialeptic seizure
-----Generalized non-motor (absence) seizure
-----Focal impaired awareness behavior arrest seizure
label: Dialeptic seizure
def: A dialeptic seizure is a type of seizure characterised predominantly by reduced responsiveness or awareness and with subsequent at least partial amnesia of the event.
comment: The main ictal manifestation of dialeptic seizures is an alteration of consciousness rather than motor or sensory features. The term describes the pure clinical semiology of a seizure and is used independently of the ictal electrographic findings: it could describe a seizure of focal, generalized or unknown onset. In 2017 the ILAE Commission for Classification and Terminology recommended classifying a seizure as having focal or generalized onset only when there is a high degree of confidence (>80%, arbitrarily chosen to parallel the usual allowable beta error) in the accuracy of this determination.
http://webprotege.stanford.edu/RiCnLhdRh6VgSt0KNJ92ib: Adapted from Cleveland 1998 but incorporating ILAE 2017 terms.
http://purl.org/dc/elements/1.1/source: PMID: 9738682
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney"
PMID:28276060: https://www.ncbi.nlm.nih.gov/pubmed/PMID:28276060

**Comment by pnrobinson** [comment]
@DLewisSmith
@galerp
@shiva-g
@ihelbig
@GBalagura
@hughkearney

There is an existing term that I think comes pretty close to the new term 'Dialeptic seizure':
https://hpo.jax.org/app/browse/term/HP:0002384
(Dialeptic seizure is listed as a related synonym, and we had actually previously obsoleted a term for 'Dialeptic seizure' because I think it was regarded as a 'fuzzy' concept). 

Can we rename this term to 'Dialeptic seizure and update the definition?

Alternatively, what is the relation to https://github.com/obophenotype/human-phenotype-ontology/issues/5307

**Comment by DLewisSmith** [comment]
Dialeptic is a fuzzy concept encompassing both 'absence seizure' (a type of generalized seizure) and 'Focal impaired awareness behavior arrest seizure’. 
It is not recognised by the current ILAE classification but important for allowing the most precise possible encoding of a non-motor seizure characterized by the person going blank and motionless in situations where the clinician does not know the electro-graphic onset of the seizure (focal or generalized).

We have also included similar terms to bridge between focal and generalized by semiology for various types of motor seizures.
